config/common_test.go

```diff
@@ -73,16 +74,6 @@ func init() {
 	}
 }
 
-func prepareExpectedConfig(t *testing.T, expectedCfg *model.Config) *model.Config {
```

Comment:
@crspeller , I miss the context, are these change and the Viper one related/prereq to this feature work or cleanup in the area?

Reply:
It's related. Basically the way we where doing the defaults for viper was causing one of my environment variable override tests to fail, but it was working overall. Me and Jesse traced it back to `set` in the common config store getting called and the overrides being applied again after set defaults. This change makes that behavior default to simplify the code and make this use case work.

Reply:
Thanks for the explanation.